### ISSUE TITLE
How do i run a client or connect as one ?

### ISSUE THREAD

**Issue by willian-ctt** [gateaubleu/Arkanic-Touch#2]
There's any way i can use the dofus lindo client to connect to it ? i got the project running, but both urls returns me nothing, any ideas ?

**Comment by bleugateau** [comment]
Yes with editing the script.js, if you need help with that leave me your Discord.
What urls not working ?

**Comment by willian-ctt** [comment]
Gladly would accept some help, my discord is
**Yeti#9065**

**Comment by bleugateau** [comment]
I added you on Discord

**Comment by Lokiitoo** [comment]
> I added you on Discord

Hi, i would like to know if you can help me with it too please ?

My discord : ~₪ Tsunamii / Sadisavage ₪~#1924
If it dosen't work give me your's i'll add you
Thank You !

**Comment by dlaudh** [comment]
My discord to talk about that 
**DualH#8105**

**Comment by bleugateau** [comment]
Add me Ten#7318 I can help you to patch your client

**Comment by Lokiitoo** [comment]
> My discord to talk about that
> **DualH#8105**



> Add me Ten#7318 I can help you to patch your client

I added both of you guys

**Comment by FioPio** [comment]
Hey, I would also like to run the client and I have no idea how to patch it, do you have any tutorial of how to do that?

**Comment by bleugateau** [comment]
Hi the version of the client is quite old, you need to update API call.
You need to patch your client from script.js and only need to update api routes.

**Comment by FioPio** [comment]
and how do I do that?

**Comment by kimbaxz** [comment]
Hello there, I have found your github repo, and I will be very interested in testing your project. I was able to run the server without too much problem in an ubuntu container.

I downloaded the Lindo source code in order to modify the "script.js" as indicated on the closed issues. However I think I missed a parameter, I have for the moment modified the url of the server with "localhost".

Do I have to change other things in this file in order to be able to access the server?

I've added you in discord (Øya#8074)

In any case, it's a very good job!

Thank you in advance for taking the time to answer me.
